src/app/errors/routes.ts

```diff
@@ -1,10 +1,15 @@
+import Loadable from 'react-loadable';
 import { Route } from '../navigation/types';
-import * as components from './components';
 
 const CATCH_ALL = '/(.*)';
 
 export const notFound: Route<undefined> = {
-  component: components.PageNotFound,
+  component: Loadable({
+    loader: () => import(/* webpackChunkName: "PageNotFound" */ './components/PageNotFound'),
+    loading: () => null,
```

Comment:
Similarly, can in indicator of loading/loaded be included?

Reply:
these error pages aren't actually used in production